src/app/shared/services/mysql.service.ts

```diff
@@ -33,12 +33,12 @@ export class MysqlService {
   constructor(private electronService: ElectronService, private ngZone: NgZone) {
     /* istanbul ignore next */
     if (this.electronService.isElectron()) {
-      this.mysql = window.require('mysql');
+      this.mysql = window.require('mysql2');
     }
   }
 
   getConnectionState() {
-    return this._connection ? this._connection.state : null;
+    return this._connection ? 'connected' : null
```

Comment:
not sure about this

Reply:
I'm researching about that and I couldn't find any other alternative, because o this version, connection state doesn;t exist.

Reply:
ok